README.md

```diff
@@ -63,43 +63,101 @@ JPlag can either be used via the CLI or directly via its Java API. For more info
 ### CLI
 *Note that the [legacy CLI](https://github.com/jplag/jplag/blob/legacy/README.md) is varying slightly.*
 
+In the new version the language can either be set with the -l parameter or as a subcommand. If both a subcommand and the -l option are specified, the subcommand will take priority.
```

Comment:
This sentence starts oddly. There is only one version when this PR is merged into dev/main.
Just start the sentence with "The language...".